apps/wpcom-block-editor/src/common/coblocks-preview-images.js

```diff
@@ -0,0 +1,81 @@
+/**
+ * External dependencies
+ */
+import { addFilter } from '@wordpress/hooks';
+import { get } from 'lodash';
+
+function getCoBlocksMasonryExampleImages( settings, name ) {
+	if ( 'coblocks/gallery-masonry' !== name ) {
+		return settings;
+	}
+
+	return {
+		...settings,
+		example: {
+			attributes: {
+				...settings.attributes,
+				images: [
+					{ url: get( 'wpcomGutenberg', 'coblocksGalleryImages[0]' ) },
```

Comment:
Since the URLs are only missing our versioned directory, and we have the plugin version already in `wpcomGutenberg`, what if we build the URLs here?

```js
function getCoBlocksMasonryExampleImages( settings, name ) {
	if ( 'coblocks/gallery-masonry' !== name ) {
		return settings;
	}

	const images = settings.example.attributes.images.map( image => {
		return {
			...image,
			url: image.url.replace( /coblocks\/dist/, `coblocks/${ wpcomGutenberg.coblocksVersion }/dist` ),
		}
	} );

	return {
		...settings,
		example: {
			...settings.example,
			attributes: {
				...settings.example.attributes,
				images,
			},
		},
	};
}
```

That way, we wouldn't need any server-side change.

Reply:
Oh, interesting idea! I immediately jumped to server-side, since we can take advantage of `staticize_subdomain` and constants (and can report what we actually _have_, instead of assuming). But, this is lighter-weight and sidesteps the need to hang more junk on `wpcomGutenberg`. I'll do this up (adding our static domains to the mix).